GoogleMeetup/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,13 +37,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     URL url = new URL("https://data.nsw.gov.au/data/dataset/0a52e6c1-bc0b-48af-8b45-d791a6d8e289/resource/f3a28eed-8c2a-437b-8ac1-2dab3cf760f9/download/covid-case-locations-20" + date + "a.json");
 
     Scanner sc = new Scanner(url.openStream());
-
-    StringBuilder sb = new StringBuilder();
+    sc.useDelimiter("\n");
     String result = new String();
+
     while(sc.hasNext()) {
-        result = result + sc.next();
+        result = result + sc.nextLine();
```

Comment:
Last time we discussed using StringBuilder for string concatenation. Using string to operate like this is slow given strings are immutable and every time this creates a new string.